<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRegistartion</class>
 <widget class="QDialog" name="CRegistartion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>580</width>
     <height>310</height>
    </rect>
   </property>
   <property name="title">
    <string>Registration</string>
   </property>
   <widget class="QLineEdit" name="userName_line">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>30</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="userPass_line">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>135</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeatPass_line">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>170</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstName_line">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>65</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_line">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>205</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="accept_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Frist name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>205</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>235</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Show password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>235</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>!(may be unsafe)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openIcon">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open item</string>
    </property>
   </widget>
   <widget class="QLabel" name="userIcon_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>150</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;Image&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>235</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose icon</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>30</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>65</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>170</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>135</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>205</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>270</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Second name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>100</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondName_line">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>100</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
